client/src/components/notebook/Controller.ts

```diff
@@ -133,13 +134,9 @@ const getCode = (doc: vscode.TextDocument) => {
   } else if (doc.languageId === "python") {
     code = wrapPython(code);
   }
-  return outputHtml ? wrapHTML5(code) : code;
+  return outputHtml ? wrapCode(code) : code;
```

Comment:
nit: as `wrapCode` will take care of the outputHtml option, how about simply call `wrapCode` so that we don't need to get settings in this file?

Reply:
Ah, good point. I'll make that change, thanks!